docs/src/main/java/org/neo4j/driver/jdbc/docs/AuraGenAIExample.java

```diff
@@ -0,0 +1,221 @@
+/*
+ * Copyright (c) 2023-2024 "Neo4j,"
+ * Neo4j Sweden AB [https://neo4j.com]
+ *
+ * This file is part of Neo4j.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.neo4j.driver.jdbc.docs;
+
+import java.io.InputStreamReader;
+import java.net.URI;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+import java.util.function.Predicate;
+
+import com.opencsv.CSVReaderBuilder;
+import org.neo4j.driver.jdbc.Neo4jDriver;
+import org.neo4j.driver.jdbc.Neo4jPreparedStatement;
+
+public final class AuraGenAIExample {
+
+	private AuraGenAIExample() {
+	}
+
+	public static void main(String... args) throws Exception {
+
+		var openAIToken = System.getenv("OPEN_AI_TOKEN");
+		if (openAIToken == null || openAIToken.isBlank()) {
+			throw new IllegalArgumentException(
+					"Please set a system environment variable named `OPEN_AI_TOKEN` containing your OpenAI token");
+		}
+
+		// Getting a connection
+		try (var con = Neo4jDriver.withSQLTranslation().fromEnv().orElseThrow()) {
+
+			var indexes = new String[] { """
+					/*+ NEO4J FORCE_CYPHER */
+					CREATE CONSTRAINT `movie-link` IF NOT EXISTS
+					FOR (n:Movie) REQUIRE n.link IS UNIQUE
+					""", """
+					/*+ NEO4J FORCE_CYPHER */
+					CREATE CONSTRAINT `genre-name` IF NOT EXISTS
+					FOR (n:Genre) REQUIRE n.name IS UNIQUE
+					""", """
+					/*+ NEO4J FORCE_CYPHER */
+					CREATE VECTOR INDEX `movie-embeddings` IF NOT EXISTS
+					FOR (n:Movie) ON (n.embedding)
+					OPTIONS {indexConfig: {
+						`vector.dimensions`: 1536,
+						`vector.similarity_function`: 'cosine'
+					}}
+					""" };
+
+			System.out.printf("# Using a simple statement several times for creating indexes:%n%n");
+			// The most simple statement class in JDBC that is there: java.sql.Statement.
```

Comment:
```suggestion
			// The most simple statement class in JDBC that exists: java.sql.Statement.
```
or `s/that exists/existing`